### PR TITLE
Button Component

### PR DESCRIPTION
**Pull request by rkapoor10**

### CODE REVIEW

File: components/buttons/buttons.css [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+}
+
+
+.btn-disable:hover{
+    cursor: no-drop;
+    background-color: var(--lightyellow-color);
+}
+
+
+/* add minus btn -------------- */
```
Comment (Raghav888) [inline components/buttons/buttons.css:45]: Good use of comment

File: components/buttons/buttons.css [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+/* ICON BUTTONS----------------------- */
+
+.btn-icon{
+    border: none;
+    border-radius: 50%;
+    display: inline-flex;
+    justify-content: center;
+    align-items: center;
+    height: 2.5rem;
```
Comment (Raghav888) [inline components/buttons/buttons.css:77]: Try to use those values also as var, it will be helpful in future projects

**Review by Raghav888 — APPROVED** [review]
Good to merge

File: components/buttons/buttons.css (hunk 2 above)
Comment (rkapoor10) [reply components/buttons/buttons.css:77]: sure...thanks

**Review by Manju-sree — COMMENTED** [review]
code looks good.. you won't add iframe?

**Comment by rkapoor10** [comment]
> code looks good.. you won't add iframe?

will add later.

**Pull request merged**